onnxruntime/core/providers/rocm/rocm_utils.cu

```diff
@@ -0,0 +1,87 @@
+#include "hip/hip_runtime.h"
```

Comment:
(nit):  move include down by others?  (several files have this above the license)

Reply:
same, will update these files.